packages/core-mobile/app/hooks/useHomeScreenHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback, useLayoutEffect } from 'react'
+import { View } from 'react-native'
+import { Link, useNavigation } from 'expo-router'
+
+const useHomeScreenHeader = (): void => {
```

Comment:
let's move these inside the `new` folder as well

Reply:
I statically configured bar buttons so this hook has been removed.